File: lab1/app/src/test/java/vsu/com/CircleTest.java
package vsu.com;

import org.junit.Assert;
import org.junit.Test;

public class CircleTest {

    @Test
    public void testCircleArea() {
        Circle circle = new Circle("White", 2.0);
        Assert.assertEquals(12.56, circle.area(), 0.01);
    }

    @Test
    public void testCirclePerimeter() {
        Circle circle = new Circle("Beige", 2.0);
        Assert.assertEquals(12.56, circle.perimeter(), 0.01);
    }
}

File: lab1/app/src/test/java/vsu/com/CircleTest.kt
package vsu.com;

import org.junit.Assert.*

import org.junit.Test;

class CircleTest {
    @Test
    fun testCircleArea() {
        val circle = Circle("White", 2.0)
        assertEquals(12.56, circle.area(), 0.01)
    }
    @Test
    fun testCirclePerimeter() {
        val circle = Circle("Beige", 2.0)
        assertEquals(12.56, circle.perimeter(), 0.01)
    }

}
